ui/app/pages/confirm-decrypt-message/confirm-decrypt-message.component.js

```diff
@@ -191,15 +192,15 @@ export default class ConfirmDecryptMessage extends Component {
         {this.renderAccountInfo()}
         <div className="request-decrypt-message__visual">
           <section>
-            {origin.icon ? (
+            {originMetadata?.icon ? (
               <img
                 className="request-decrypt-message__visual-identicon"
-                src={origin.icon}
+                src={originMetadata?.icon}
```

Comment:
nit: useless optional chaining

Reply:
Thanks! This is fixed here: https://github.com/MetaMask/metamask-extension/pull/10256